.github/workflows/stale.yaml

```diff
@@ -0,0 +1,42 @@
+name: 'Close stale issues and PRs'
+
+on:
+  push:
+    paths:
+      - .github/workflows/stale.yml
+    branches-ignore:
+      - dependabot/**
+  schedule:
+    # Once every day at midnight UTC. Source: https://crontab.guru/every-day
+    - cron: '0 0 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          # Setting this variable to 'true' will cause the
+          # action to run in a dry-run mode
+          debug-only: ${{ vars.DEBUG_ONLY == 'true' }}
+          stale-issue-message: >
+            This issue has been automatically marked as stale because it has not had
+            recent activity. It will be closed if no further activity occurs.
+          close-issue-message: >
+            Closing this issue after a prolonged period of inactivity. If this
+            issue is still present in the latest release, please feel free to
```

Comment:
Do we want to also mention `reopening` the same issue?

Reply:
I feel we shouldn't. As the stale state itself is reached after 30 days, then the issue gets closed after 7 days. It's already too long of a time with a reminder. Also, it's always better to create a new issues which will ensure visibility with anyone interested in recent issues, instead of [_necro-bumping_](https://communitybuilding.stackexchange.com/questions/2632/why-do-internet-forums-tend-to-prohibit-responding-to-inactive-threads).

On an afterthought, we could also lock a conversation before closing (there's an action for that 😆 https://github.com/dessant/lock-threads). I've seen locking of threads in other open source projects.

But if you feel I should update the text, I can do that. Let me know your thoughts.

Reply:
> it's always better to create a new issues which will ensure visibility with anyone interested in recent issues, instead of [necro-bumping](https://communitybuilding.stackexchange.com/questions/2632/why-do-internet-forums-tend-to-prohibit-responding-to-inactive-threads)

Nice post, helps in understanding the problems we might face if we allow older issues to get into active state.

> we could also lock a conversation before closing

I don't see much value in locking the issue/PR and keeping it open.

Reply:
Let's keep these settings as is and we might update it later if needed.